supporting_content/columns/negotiatedlistunitprice.md

```diff
@@ -0,0 +1,64 @@
+| Microsoft | Cost details                 | Not available?<br>**unitPrice** is probably the closest match but not applicable in all use cases<br>*see Discussion / Scratch space - Microsoft column mappings related issues for more details* |
+| OCI       | Cost and Usage Report        | **cost/unitPrice** is the best match<br>*see Discussion / Scratch space - OCI column mappings related issues for more details* |
+
+## References and Resources
+
+### AWS
+
+* [Line item details - AWS Cost and Usage Reports](https://docs.aws.amazon.com/cur/latest/userguide/Lineitem-columns.html)
+
+### GCP
+
+* [Structure of Detailed data export | Cloud Billing](https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage)
+* [Structure of pricing data export | Cloud Billing](https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/pricing-data)
+* [Get Google Cloud pricing information | Cloud Billing (GCP Cloud Billing API)](https://cloud.google.com/billing/docs/how-to/get-pricing-information-api)
+
+### Microsoft
+
+* [Understand usage details fields - Microsoft Cost Management](https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields)
+* [Azure Billing Enterprise APIs - PriceSheet | Microsoft Learn](https://learn.microsoft.com/en-us/rest/api/billing/enterprise/billing-enterprise-api-pricesheet)
+  * Applies only to EA
+  * *TODO: Check if there is a Microsoft Custom Prices REST API, which might provide custom prices for all account types*
+
+### OCI
+
+* [Cost and Usage Reports Overview](https://docs.oracle.com/en-us/iaas/Content/Billing/Concepts/usagereportsoverview.htm)
+
+## Example usage scenarios
+
+* *TODO*
+
+## Discussion / Scratch space
+
+### GCP column mappings related issues
+
+* **price.effective_price** is present only in detail export and thus available only for small subset of services/resources. In order to resolve this information we should rely on **BigQuery Pricing Data Export** and/or **Cloud Billing API**, which provide price related structs (out-of-scope of this document).
+* Similar to cost, regardless of the data source we use, the resolved price does not include reductions provided in credits, which aligns with the definition of NegotiatedListUnitPrice.
+
+### Microsoft column mappings related issues
+
+* [Azure Billing Enterprise APIs - PriceSheet | Microsoft Learn](https://learn.microsoft.com/en-us/rest/api/billing/enterprise/ provides information for EA.
+* TODO: Check if there is a Microsoft Custom Prices REST API, which might provide custom/negotiated prices for all account types.
+* **unitPrice** (The price per unit for the charge.) includes commitment based discounts and thus is not applicable for all use cases. Furthermore,  
```

Comment:
To my understanding, for Azure EA, the UnitPrice excludes commitment-based discounts, and so it can be used to calculate the (negotiated) cost without the commitment discounts (Microsoft documentation calls it the "pay-as-you-go cost").

https://learn.microsoft.com/en-us/azure/cost-management-billing/reservations/understand-reserved-instance-usage-ea#determine-reservation-savings
> Multiply the UnitPrice value with Quantity values to get estimated pay-as-you-go costs, if the reservation discount didn't apply to the usage.

https://learn.microsoft.com/en-us/azure/cost-management-billing/savings-plan/utilization-cost-reports#determine-savings-resulting-from-savings-plan
>  Multiply the UnitPrice value with Quantity values to get estimated pay-as-you-go costs if the savings plan discount didn't apply to the usage.